docs/how-to/extending/sql.qmd

```diff
@@ -177,7 +177,7 @@ with closing(con.raw_sql("CREATE TEMP TABLE my_table AS SELECT * FROM RANGE(10)"
 
 Here's an example:
 
-```{python}
+```python
```

Comment:
I need to revisit the return of `raw_sql` for duckdb, it doesn't seem like it can ever be closed when using raw `duckdb`.